mpeg2dec.c

```diff
@@ -1685,25 +1685,19 @@ int stream_component_open(VideoState *is, int stream_index)
 #endif
         }
 
-        if (codecCtx->codec_id == AV_CODEC_ID_H264) {
+        if ((!use_cuvid) && (codecCtx->codec_id == AV_CODEC_ID_H264)) {
+        	// SW decoder for h264, cannot resize to lower resolution
             is_h264 = 1;
 #ifdef DONATOR
 #else
-            Debug(0, "h.264 video can only be processed at full speed by the Donator version\n");

```

Comment:
No need to change this

Reply:
That's fine, I can undo it. It was just to clarify, confused by me initially (as it's not speed that is full, rather resolution). But I can revert, NP.